### ISSUE TITLE
Script Error: No such export isPed in resource isPed

### ISSUE THREAD

**Issue by christerjohansson** [STR0UDY/wf_alerts#3]
I get a lot of errors when loading up this scripts.
Error says,
SCRIPT ERROR: citizen/scripting/lua/scheduler.lua:902: No such export isPed in resource isPed.
fn (@wf_alerts/client/cl_main.lua:68

I am using FxServer version 2430, with ESX 1.2 Final. (latest available scripts for ESX)

**Comment by JamsRepos** [comment]
Change the line where it finds the job for the ESX equivalent?

**Comment by christerjohansson** [comment]
Ok, provide a solutiion please. :)

**Comment by JamsRepos** [comment]
> Ok, provide a solutiion please. :)

Nope, I don't use ESX and I'm not going to look up how to integrate collecting a job for you. There's extensive documentation out there.

**Comment by SW1FT** [comment]
@christerjohansson https://gist.github.com/SW1FT/36e5becd6e08465cb3629d6dd87f5c47
